src/store/actions.ts

```diff
@@ -443,6 +443,9 @@ export const checkoutNameRequest: ActionIF = async ({ commit, getters }): Promis
 
 export const checkinNameRequest = async ({ getters }): Promise<boolean> => {
   try {
+    // Return if approved Name Request because it was never checked out
+    if (getters.getNrState === NameState.APPROVED) return true
```

Comment:
Does an approved NR get checked out if I edit it?

Reply:
It doesn't actually, just learned that as well. Seems like thats because you are unable to edit anything beyond applicant info.

Reply:
Should it get checked out, to prevent concurrent changes? Or does that cause greater problems at this time?

Would you mind putting in a more details comment to explain the current thinking? A future developer would probably love to know the business/technical logic/decision behind this.

Reply:
That is a question i don't think i can answer. 
My understanding of the WHY on this whole check in / check out process is pretty high level. 
...To prevent changes happening on client side / staff side at the same time i imagine, although i think Thor already mentioned a mechanism that would prevent that. So i'm really not sure 🤔 

I will certainly expand the comment though.

Reply:
Looks good.